scripts/move_wcs.py

```diff
@@ -0,0 +1,97 @@
+import glob
+from astropy.io import fits
+from astropy import wcs
+from astropy.wcs import InvalidTransformError
+import six
+
+"""
+More Kw to consider 
+radesys - I think it should be moved - "OTHER" in HST
+velocity aberrration related kw - move them
+  DVA_RA< DVA_DEC, VA_SCALE
+
+TARG_RA, TARG_DEC
+"""
+stsci_wcs_kw = ['PA_V3', 'V2_REF', 'V3_REF', 'PA_APER', 'VPARITY',
+                'RA_V1', 'DEC_V1', 'ROLL_REF', 'RA_REF', 'DEC_REF']
+
+wcslib_kw_to_remove = ['LONPOLE', 'LATPOLE', 'MJD-OBS', 'DATE-OBS']
+
+
+def move_wcs(files):
+    for name in files:
+        print('Working on file {0}.'.format(name))
+        f = fits.open(name, mode='update')
+        if f[0].header['telescop'] != 'JWST':
+            f.close()
+            continue
+
+        new_hdr = _collect_wcs_keywords(f)
+        update_sci_extension(f, new_hdr)
+        clean_primary_header(f, new_hdr)
+        f.close()
+    
+    
+def _collect_wcs_keywords(f):
+    # Convert the WCS to a header
+    try:
+        w = wcs.WCS(f[0].header)
+    except InvalidTransformError:
+        if f[0].header['cunit3'] and f[0].header['cunit3'] == 'micron':
+            f[0].header['cunit3'] = 'um'
+            w = wcs.WCS(f[0].header)
+    new_hdr = w.to_header()
```

Comment:
The to_header() method does not include elements of the PCn_m matrix if they are equal to the unit matrix, which many of our test data have. Hence the PCn_m keywords do not appear in the SCI header, nor are they removed from the primary header (because only the keywords in new_hdr are removed).